change/@fluentui-react-accordion-a88290b3-b03a-40b9-a8cc-d24ab80c5071.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fixup! fix(react-accordion): fix missing dependencies",
```

```suggestion
  "comment": "add missing dependencies",
```

_nit_ "fixup!" should not be included, right? We also don't include package names as the changelog is grouped anyway

---

2x yes ! provided context below